Add SIG Community Charter (#229)
cilium/community: sig-community/README.md

# SIG Community

> SIG Community is responsible for fostering, growing, and sustaining the Cilium open source community. This includes contributor experience, community engagement, governance updates, and advocacy efforts.

Author: ```suggestion
SIG Community is responsible for fostering, growing, and sustaining the Cilium open source community. This includes contributor experience, community engagement, recommending governance updates, and advocacy efforts.
```

The [charter](charter.md) defines the scope and governance.

## Meetings and Channel

> Join our [Slack channel](TBD) in the Cilium Slack, or join our meetings:  

Author: ```suggestion
Join our [Slack channel](https://cilium.slack.com/archives/C08MR0FQRCM) in the Cilium Slack, or join our meetings:  
```

> * Monthly on the [TBD] - [Meeting Notes](TBD)

Author: ```suggestion
* Monthly on the second Thursday at 8am PT - [Meeting Notes](TBD)
```

# SIG Community

SIG Community is responsible for fostering, growing, and sustaining the Cilium open source community. This includes contributor experience, community engagement, recommending governance updates, and advocacy efforts.

The [charter](charter.md) defines the scope and governance.

## Meetings and Channel

Join our [Slack channel](https://cilium.slack.com/archives/C08MR0FQRCM) in the Cilium Slack, or join our meetings:  
* Monthly on the second Thursday at 8am PT
 - [Zoom link](https://zoom-lfx.platform.linuxfoundation.org/meeting/91921107999?password=8cf3a7e1-56ea-484e-9c1a-3c7bf26bfbf3)
 - [Meeting Notes](https://docs.google.com/document/d/1QykQGWD_XjxWBKfGhgyk2hrOd7pq1kZ1QRh0Aoyn7PU/edit?usp=sharing)

## Leadership

### Leads

The Leads are responsible for driving the operations and community initiatives of SIG Community, as well as upholding the [community values](https://github.com/cilium/community/blob/main/VALUES.md).  
* **[Bill Mulligan]** (@xmulligan)  

> * **[Bowei Du]** (@bowei)

Author: ```suggestion
* **[Bowei Du]** (@bowei)
* **[Tamilmani Manoharan]** (@tamilmani1989)

```
